src/TortoiseProc/GitLogListBase.cpp

```diff
@@ -3859,6 +3859,70 @@ void CGitLogListBase::OnFind()
 		m_pFindDialog->Create(this);
 	}
 }
+void CGitLogListBase::JumpToGitHash(CString &hash, bool bClearFullSelection)
+{
+	for (int i = 0; i < m_arShownList.GetCount(); ++i)
+	{
+		GitRev *rev = (GitRev *)m_arShownList.SafeGetAt(i);
+		if (!rev) continue;
+		if (rev->m_CommitHash.ToString().Left(hash.GetLength()) == hash)
+		{
+			int iPreviousFocus = -1;
+			if (bClearFullSelection)
+			{
+				// work around bug in which the previously focused item, if not selected, would not be redrawn to erase its focus rectangle

```

Comment:
I tried, but did not found why this workaround is needed. Can you provide some steps to trigger this (or where without this workaroud sopmething bad happens)=